content/ADR-239-avatar-assembly-changes.md

```diff
@@ -0,0 +1,197 @@
+---
+adr: 239
+date: 2023-06-01
+title: Avatar Assembly Changes
+status: Review
```

Comment:
This PR has a tide effect of potentially dozens/hundreds of hours across many teams.

Please update https://adr.decentraland.org/adr/ADR-57 (Avatar assembling instructions for visualization) to reflect the new changes in this same PR. That way we can use the diff in the previous one to update the:
- bevy implementation https://github.com/decentraland/bevy-explorer/pull/19/files
- babylon implementation https://github.com/decentraland/hammurabi/pull/31
- werable preview for marketplace & for third parties (DG mobile) https://wearable-preview.decentraland.org/?profile=default1
- docs site preview https://docs.decentraland.org/creator/emotes/emotes-overview/

Reply:
I thought old ADRs were supposed to be set in stone once approved, should I modify that ADR in this PR? Or how should I handle this? @menduz

Reply:
done